uitests/tests/org.jboss.tools.fuse.ui.bot.tests/resources/routeWithValidationFixedOnGlobalElements.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans"
```

Comment:
why this file is part of this PR? it seems strange

Reply:
Good question. I didn't change that file.